WordPress/Classes/ViewRelated/Post/PostListFilterSettings.swift

```diff
@@ -0,0 +1,166 @@
+import Foundation
+import WordPressComAnalytics
+
+@objc class PostListFilterSettings: NSObject {
```

Comment:
Also: can you add documentation to the class to explain why it exists / what it does?